ospd/scan.py

```diff
@@ -56,13 +58,14 @@ class ScanCollection:
 
     """
 
-    def __init__(self) -> None:
+    def __init__(self, file_storage_dir) -> None:
```

Comment:
Please always add type hints
```suggestion
    def __init__(self, file_storage_dir: str) -> None:
```